modules/acme_bucket/main.tf

```diff
@@ -4,17 +4,10 @@ data "aws_s3_bucket" "logging_bucket" {
 
 resource "aws_s3_bucket" "acme_bucket" {
   bucket = var.bucket_name
-
+  acl    = "public-read"
```

Comment:
tfsec check aws-s3-no-public-access-with-acl failed. 

Resource 'module.acme_finance_bucket:aws_s3_bucket.acme_bucket' has an ACL which allows public access.

For more information, see:

- https://aquasecurity.github.io/tfsec/latest/checks/aws/s3/no-public-access-with-acl
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket
- https://aws.amazon.com/premiumsupport/knowledge-center/secure-s3-resources/